src/analysis/normed/field/basic.lean

```diff
@@ -481,6 +481,16 @@ begin
   simp,
 end
 
+lemma roots_of_unity.norm_one [comm_monoid β] (φ : β →* α) {x : β} {k : ℕ+}
```

Comment:
Now the name should be changed, since the theorem doesn't mention `root_of_unity`...

Reply:
``nth_roots.norm_one```? ```norm_one_of_pow_eq_one```? 

I like the last one a bit better... but you might have a better suggestion.

Reply:
I agree the last one is better.